src/Paprika.Runner/Program.cs

```diff
@@ -23,7 +23,7 @@ public static class Program
 
     private const int NumberOfLogs = 10;
 
-    private const long DbFileSize = 10 * Gb;
+    private const long DbFileSize = 6 * Gb;
```

Comment:
why not read it from arguments?

Reply:
Will be squeezing more, the further we go. And yes, later, this will be automatically set by the db.